src/sentry/static/sentry/app/views/teamDetails.jsx

```diff
@@ -89,33 +87,20 @@ const TeamDetails = createReactClass({
     else if (!team || this.state.error) return <LoadingError onRetry={this.fetchData} />;
 
     let routePrefix = recreateRoute('', {routes, params, stepBack: -1}); //`/organizations/${orgId}/teams/${teamId}`;
-    let access = this.getAccess();
 
     //TODO(maxbittker) remove hack to not show this page on old settings
     let onNewSettings = routePrefix.startsWith('/settings/');
 
-    const features = new Set(this.context.organization.features);
     return (
       <div>
         <h3>{team.name}</h3>
 
-        {!features.has('new-settings') &&
```

Comment:
Doesn't this mean for all users, they won't be able to remove a team?

iirc Remove Team in Team Settings is only for users with `new-settings` feature tag.

Reply:
yeah this is weird. i'm not sure why this dropdown was even appearing for me, but I will figure out what is going on and resolve it immediately

Reply:
(my assumption was that is was a leftover feature duplication from before we added the delete button to team settings)